e2e/pages/citizen/createCase/C100/otherProceedings/EmergencyProtectionDocumentUploadPage.ts

```diff
@@ -128,17 +126,19 @@ export class EmergencyProtectionDocumentUploadPage {
   private static async fillInFields({
     page: page,
   }: fillInFieldsOptions): Promise<void> {
-    const fileInput = page.locator(`${UniqueSelectors.documentUpload}`);
+    const fileInput = page.locator(
+      `${CitizenOtherProceedingsDocumentUploadSelectors.documentUpload}`,
+    );
     await fileInput.setInputFiles(config.testPdfFile);
     await page.click(
       `${Selectors.GovukButton}:text-is("${CommonStaticText.uploadFile}")`,
     );
     await page.waitForSelector(
-      `${UniqueSelectors.uploadConfirmationSelector}`,
+      `${CitizenOtherProceedingsDocumentUploadSelectors.uploadConfirmationSelector}`,
       { timeout: 5000 },
     );
     const isUploaded = await page.isVisible(
-      `${UniqueSelectors.uploadConfirmationSelector}`,
+      `${CitizenOtherProceedingsDocumentUploadSelectors.uploadConfirmationSelector}`,
```

Comment:
Could we turn the file upload validations into a Helper like we did with the solicitor journey? Should be easier and more consistent down the line